File: javascript/conduits/setConduits.js
import { PWRWallet } from "@pwrjs/core";
import dotenv from 'dotenv';
dotenv.config();

// Setting up your wallet in the SDK
const privateKey = process.env.PRIVATE_KEY;
const wallet = new PWRWallet(privateKey);

async function conduits() {
    const conduits = [
        Buffer.from("conduit_node_address", "hex"),
    ];
    const vmId = 9999;

    const res = await wallet.setConduits(vmId, conduits);
    console.log(res.transactionHash);
}
conduits();

File: python/conduits/set_conduits.py
from pwrpy.pwrwallet import PWRWallet
from dotenv import load_dotenv
import os
load_dotenv()

# Setting up your wallet in the SDK
private_key = os.getenv("PRIVATE_KEY")
wallet = PWRWallet(private_key)

def conduits():
    conduits = [
        bytes.fromhex("conduit_node_address"),
    ]
    vm_id = 9990

    # Sending the VM data transaction
    res = wallet.set_conduits(vm_id, conduits)
    print(res.data)

conduits()